packages/es-components/src/components/containers/tabPanels/TabPanel.js

```diff
@@ -100,7 +113,8 @@ TabPanel.propTypes = {
   /**
    * Makes sure immediate children are Tab or Tab List, as we cannot render anything else in the tab heading.
    */
-  children: childrenRule
+  children: childrenRule,
+  className: PropTypes.string
```

Comment:
Do we need to add a className prop if we are adding some classnames that can act as handles?